app/controllers/idv/sessions_controller.rb

```diff
@@ -31,6 +36,19 @@ def update
 
     private
 
+    def validation_errors
+      @_validation_errors ||= []
```

Comment:
As this article says, "keep business logic out of the controller and views": https://robots.thoughtbot.com/activemodel-form-objects